### PR TITLE
Adding separate EE9 api/api bundle projects

### PR DESCRIPTION
**Pull request by cbridgha**

### CODE REVIEW

**Comment by cbridgha** [comment]
#build

File: dev/com.ibm.websphere.appserver.api.authData.ee9/bnd.bnd [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+-includeresource: {META-INF/maven/com.ibm.websphere.appserver.api/com.ibm.websphere.appserver.api.authData.ee9.jakarta/pom.xml=com.ibm.websphere.appserver.api.authData.ee9.jakarta.pom}
+
+publish.wlp.jar.suffix: dev/api/ibm
+publish.wlp.jar.exclude: com.ibm.websphere.appserver.api.authData.ee9.jar
+
+-buildpath: \
+	com.ibm.ws.crypto.passwordutil;version=latest, \
+	com.ibm.ws.security.auth.data;version=latest, \
+	com.ibm.ws.security.auth.data.common;version=latest
```
Comment (tevans78) [inline dev/com.ibm.websphere.appserver.api.authData.ee9/bnd.bnd]:
```suggestion
Bundle-SymbolicName: com.ibm.websphere.appserver.api.authData.jakarta

Import-Package: com.ibm.websphere.crypto,com.ibm.websphere.security.auth.data,com.ibm.websphere.security.jca

Export-Package: com.ibm.websphere.crypto,com.ibm.websphere.security.auth.data,com.ibm.websphere.security.jca

-includeresource: {META-INF/maven/com.ibm.websphere.appserver.api/com.ibm.websphere.appserver.api.authData.ee9.jakarta/pom.xml=com.ibm.websphere.appserver.api.authData.ee9.jakarta.pom}

publish.wlp.jar.suffix: dev/api/ibm
#I don't understand why the new directive is needed if this bundle doesn't get included in a feature
#publish.wlp.jar.exclude: com.ibm.websphere.appserver.api.authData.ee9.jar

-buildpath: \
	com.ibm.ws.crypto.passwordutil.jakarta;version=latest, \
	com.ibm.ws.security.auth.data.jakarta;version=latest, \
	com.ibm.ws.security.auth.data.common.jakarta;version=latest
```

Comment (tevans78) [reply dev/com.ibm.websphere.appserver.api.authData.ee9/bnd.bnd]: not sure I got the includeresource bit right

**Comment by cbridgha** [comment]
#build

File: dev/com.ibm.websphere.appserver.api.authData.ee9/bnd.bnd (hunk 1 above)
Comment (cbridgha) [reply dev/com.ibm.websphere.appserver.api.authData.ee9/bnd.bnd]: yeah I removed all transformations....

**Review by jhanders34 — DISMISSED** [review]

**Comment by cbridgha** [comment]
#build

**Pull request merged**